SmartDeviceLinkTests/DevAPISpecs/SDLPreloadChoicesOperationSpec.m

```diff
@@ -297,6 +308,121 @@
                 });
             });
         });
+
+        context(@"updating choices", ^{
+            __block SDLChoiceCell *testCell1 = nil;
+            __block SDLChoiceCell *testCell2 = nil;
+            __block NSOrderedSet<SDLChoiceCell *> *testCells = nil;
+
+            beforeEach(^{
+                testCell1 = [[SDLChoiceCell alloc] initWithText:@"Cell1" artwork:nil voiceCommands:nil];
+                testCell2 = [[SDLChoiceCell alloc] initWithText:@"Cell2" secondaryText:nil tertiaryText:nil voiceCommands:nil artwork:nil secondaryArtwork:[SDLArtwork artworkWithStaticIcon:SDLStaticIconNameClock]];
+                testCells = [[NSOrderedSet alloc] initWithArray:@[testCell1, testCell2]];
+            });
+
+            context(@"if a choice item is removed", ^{
+                it(@"should be removed if the removal is attempted while the operation is pending", ^{
+                    SDLTextField *primaryTextField = [[SDLTextField alloc] init];
+                    primaryTextField.name = SDLTextFieldNameMenuName;
+                    windowCapability.textFields = @[primaryTextField];
```

Comment:
I think that this can be moved from this it and the below to the above context

Reply:
I removed the code as it was unnecessary.